### ISSUE TITLE
Include accessibility statement on dashboard and replace "Report an Issue" with "API Docs" 

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#440]
@alexstine reported:

> Could we please put the accessibility statement on the main Equalify website? Read the Accessibility Statement footer link while logged in currently points to: https://github.com/EqualifyEverything/v1/blob/main/ACCESSIBILITY.md. I also think it is super unreasonable to ask people with disabilities having trouble with Equalify to open GitHub issues, this process should also be offered via email, contact form, etc. We're using WordPress and Gravity Forms, should be a 10min job

Can we add the accessibility statement into the dashboard?

While we're at it, we should replace "Report and Issue" with "[API Docs](https://docs.equalify.app)".

Here is the updated accessibility statement:

```
# Equalify Accessibility Statement

## How Can We Help?
We welcome any comments, questions, or feedback on our site. If you notice aspects of our site that aren’t working for you or your assistive technology, email [support@equalify.app](mailto:support@equalify.app).

## Equalify is Committed to Digital Accessibility
This project is committed to delivering an excellent user experience for everyone. Equalify's user interface is structured in a way that allows those of all abilities to easily and quickly find the information they need.

## Ongoing Efforts to Ensure Accessible Content
Equalify uses the Web Content Accessibility Guidelines (WCAG) version 2.2 as its guiding principle. As we develop new pages and functionality, the principles of accessible design and development are an integral part of conception and realization.

We continually test content and features for WCAG 2.2 Level AA compliance and remediate any issues to ensure we meet or exceed the standards. Testing of our digital content is performed by our accessibility experts using automated testing software, screen readers, a color contrast analyzer, and keyboard-only navigation techniques.

## Summary of Accessibility Features
- All images and other non-text elements have alternative text associated with them.
- Navigational aids are provided on all app pages. 
- Structural markup to indicate headings and lists has been provided to aid in page comprehension
- Forms are associated with labels and instructions on filling in forms are available to screen reader users

## Project VPATs
To request a conformance report using the Voluntary Product Accessibility Template (VPAT), please email [support@equalify.app](mailto:support@equalify.app).
```

**Comment by bbertucc** [comment]
@azdak assigning this to you. Updated the issue with the accessibility statement and an additional footer link

**Comment by bbertucc** [comment]
This is on staging. @alexstine does the Accesssibility statement look good on http://equalify.dev?

**Comment by alexstine** [comment]
@azdak Could you please update the page announcer routes?
1. Go to https://equalify.dev/accessibility/
2. Notice how this message is still displayed.
> Reports page loaded

Thanks.

**Comment by azdak** [comment]
@alexstine whoops- should be fixed in https://github.com/EqualifyEverything/equalify-dashboard/commit/cd9012c71c67a7dff8ca09cd74150a9d4846676f!

**Comment by alexstine** [comment]
@azdak Perfect.

**Comment by bbertucc** [comment]
@bbertucc will finalize and push

**Comment by bbertucc** [comment]
Up and ready
